src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -352,25 +272,15 @@ def forward(
         use_cache=False,
         output_attentions=False,
     ):
-        # hidden_states: [batch_size, seq_length, hidden_size]
-        # repeat alibi tensor with the batch size
-        alibi = alibi.repeat(hidden_states.shape[0], 1, 1).to(hidden_states.device)
-
-        # apply preprocessing if the input is padded
-        if attention_mask is not None and 0 in attention_mask:
-            alibi = pre_process_alibi_for_pad(alibi, attention_mask, self.num_heads)
-
-        mixed_x_layer = self.query_key_value(hidden_states)
-
-        # [batch_size, seq_length, 3 x hidden_size] --> [batch_size, seq_length, num_heads, 3 x head_dim]
-        new_tensor_shape = mixed_x_layer.size()[:-1] + (self.num_heads, 3 * self.head_dim)
-        mixed_x_layer = mixed_x_layer.view(*new_tensor_shape)
+        alibi = alibi.to(hidden_states.device)  # to make the model possible to run under accelerate
```

Comment:
Should alibi have a default None then?

Reply:
Nope, that's a nit as well ! should not be default to None